applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.cpp

```diff
@@ -0,0 +1,1783 @@
+    Properties::Pointer pProperties)
+    : SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>(NewId, pGeometry, pProperties) {}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::~SymbolicDConvectionDiffusionExplicit() {}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Element::Pointer SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Create(
+    IndexType NewId,
+    NodesArrayType const& ThisNodes,
+    Properties::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, this->GetGeometry().Create(ThisNodes), pProperties);
+}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Element::Pointer SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Create(
+    IndexType NewId,
+    GeometryType::Pointer pGeom,
+    Properties::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, pGeom, pProperties);
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateLocalSystem(
+    MatrixType& rLeftHandSideMatrix,
+    VectorType& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    const auto& r_geometry = this->GetGeometry();
+    const unsigned int local_size = r_geometry.size();
+
+    // Resize and intialize output
+    if (rLeftHandSideMatrix.size1() != local_size)
+        rLeftHandSideMatrix.resize(local_size, local_size, false);
+    if (rRightHandSideVector.size() != local_size)
+        rRightHandSideVector.resize(local_size, false);
+    noalias(rLeftHandSideMatrix) = ZeroMatrix(local_size, local_size);
+    noalias(rRightHandSideVector) = ZeroVector(local_size);
+
+    // Element variables
+    ElementVariables rVariables;
+    this->InitializeEulerianElement(rVariables,rCurrentProcessInfo);
+
+    // Compute tau
+    this->CalculateTau(rVariables);
+
+    // Execute RHS-LHS build
+    this->CalculateLocalSystemInternal(rVariables,rLeftHandSideMatrix,rRightHandSideVector);
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateRightHandSide(
```

Comment:
Shouldn't this method be exactly the same as in the QS base class?

Reply:
As commented above, I could not derive it from QS base class. I resolve the conversation since such method is removed. Feel free to re-open it for discussion.